test/cog/chat/slack/templates/embedded/bundle_install_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Cog.Chat.Slack.Templates.Embedded.BundleInstallTest do
+  use Cog.TemplateCase
+
+  test "bundle-info template" do
+    data = %{"results" => [%{"name" => "heroku",
+                             "version" => "0.0.4"}]}
+
+    expected = """
+    Bundle "heroku" version "0.0.4" installed.
+    """ |> String.strip
```

Comment:
For short-and-sweet templates like this, a single line is fine, instead of the whole triple-quote-and-strip dance.